openstack/cinder/templates/etc/_cinder-policy.yaml.tpl

```diff
@@ -0,0 +1,142 @@
+"volume_extension:volume_admin_actions:migrate_volume_completion": "rule:context_is_admin"
+
+"volume_extension:volume_host_attribute": "rule:context_is_admin"
+"volume_extension:volume_tenant_attribute": "rule:context_is_viewer"
+"volume_extension:volume_mig_status_attribute": "rule:context_is_admin"
+"volume_extension:hosts": "rule:context_is_admin"
+"volume_extension:services:index": "rule:context_is_admin"
+"volume_extension:services:update" : "rule:context_is_admin"
+
+"volume_extension:volume_manage": "rule:context_is_admin"
+"volume_extension:volume_unmanage": "rule:context_is_admin"
+
+"volume_extension:capabilities": "rule:context_is_admin"
+
+"volume:create_transfer": "rule:context_is_editor"
+"volume:accept_transfer": "rule:edit_all"
+"volume:delete_transfer": "rule:context_is_editor"
+"volume:get_all_transfers": "rule:view_all"
+
+"volume_extension:replication:promote": "rule:context_is_admin"
+"volume_extension:replication:reenable": "rule:context_is_admin"
+
+"volume:enable_replication": "rule:context_is_admin"
+"volume:disable_replication": "rule:context_is_admin"
+"volume:failover_replication": "rule:context_is_admin"
+"volume:list_replication_targets": "rule:context_is_admin"
+
+"backup:create" : "rule:context_is_editor"
+"backup:delete": "rule:context_is_editor"
+"backup:get": "rule:context_is_editor"
+"backup:get_all": "rule:context_is_editor"
+"backup:restore": "rule:context_is_editor"
+"backup:backup-import": "rule:context_is_editor"
+"backup:export-import": "rule:context_is_editor"
+
+"snapshot_extension:snapshot_actions:update_snapshot_status": "rule:context_is_admin"
+"snapshot_extension:snapshot_manage": "rule:context_is_admin"
+"snapshot_extension:snapshot_unmanage": "rule:context_is_admin"
+
+# Groups (not yet supported in ccloud)
+# So forcing these all to admin only
+"group:create": "rule:context_is_admin"
+"group:delete": "rule:context_is_admin"
+"group:get" :   "rule:context_is_admin"
+"group:get_all" :   "rule:context_is_admin"
+"group:update" :   "rule:context_is_admin"
+"group:reset_status" :   "rule:context_is_admin"
+"group:group_project_attribute" :   "rule:context_is_admin"
+"group:group_types_manage" :   "rule:context_is_admin"
+"group:group_types_specs" :   "rule:context_is_admin"
+"group:access_group_types_specs" :   "rule:context_is_admin"
+"group:get_all_group_snapshots" :   "rule:context_is_admin"
+"group:create_group_snapshot" :   "rule:context_is_admin"
+"group:get_group_snapshot" :   "rule:context_is_admin"
+"group:delete_group_snapshot" :   "rule:context_is_admin"
+"group:update_group_snapshot" :   "rule:context_is_admin"
+"group:group_snapshot_project_attribute" :   "rule:context_is_admin"
+"group:reset_group_snapshot_status" :   "rule:context_is_admin"
+"group:enable_replication" :   "rule:context_is_admin"
+"group:disable_replication" :   "rule:context_is_admin"
+"group:failover_replication" :   "rule:context_is_admin"
+"group:list_replication_targets" :   "rule:context_is_admin"
```

Comment:
hm ... I'm annoyed by those lines having a space before the `:`. We don't do it everywhere in the file. Can we straighten this out with a small run of `sed -r 's/" +:/":/' _cinder-policy.yaml.tpl`, please?